Orange/widgets/data/owfile.py

```diff
@@ -404,6 +406,30 @@ def get_ext_name(filename):
 
         self.report_data("Data", self.data)
 
+    def dragEnterEvent(self, event):
+        """Accept drops of valid file urls"""
+        def _is_osx_file_reference_url():
+            for url in event.mimeData().urls():
+                local_url = url.toLocalFile()
+                return local_url.startswith('/.file/id=')
+
+        if event.mimeData().hasUrls():
+            # On OSX Yosemite and later, dragged files are passed
+            # as file references (file://.file/id=... urls).
+            # As PyQt4 toLocalFile does not resolve such urls,
+            # we do not allow them to be dropped on widget.
+            if _is_osx_file_reference_url():
+                return
+
+            event.acceptProposedAction()
```

Comment:
Before accepting, it should probably ensure there exists a reader for the associated file. Otherwise if breaks at least on ZIP archives when I press _Reload_.